data/Pakistan/Assembly/sources/manual/group_wikidata.csv

```diff
@@ -1,24 +1,4 @@
 id,wikidata
```

Comment:
Removing these (without explanatory comment) in https://github.com/everypolitician/everypolitician-data/commit/8dfc291e8052e7ae228e2c817b2721ee71c084e9 is a little confusing. I think I'd start this by reverting #88178 (probably as a separate PR), now that we're not going that route any more.